pages/guideforms.py

```diff
@@ -71,6 +95,24 @@ def validate(self, value):
     'pattern': EMAIL_ADDRESSES_REGEX
 }
 
+# From https://rodneyrehm.de/t/url-regex.html#imme_emosol+ht-%26f-tp%28s%29
+URL_REGEX = '[ ]*(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?[ ]*'
+# Multiple URLs, one per line
+MULTI_URL_REGEX = URL_REGEX + '(\\n' + URL_REGEX + ')*'
+
+URL_FIELD_ATTRS = {
+    'title': 'Enter a full URL https://...',
+    'placeholder': 'https://...',
+    'pattern': URL_REGEX
+}
+
+MULTI_URL_FIELD_ATTRS = {
+    'title': 'Enter one or more full URLs, one per line:\nhttps://...\nhttps://...',
+    'placeholder': 'https://...\nhttps://...',
+    'rows': 4, 'cols': 50, 'maxlength': 500
```

Comment:
Let's bump up the maxlength to 5000.  I don't know why it was set so small before.

Reply:
Sounds good.  I was wondering about that.  Is there any limitation on the database side?